### PR TITLE
feat(newserver): Jedi Knight II: Jedi Outcast

### PR DESCRIPTION
**Pull request by dgibbs64**

### CODE REVIEW

**Comment by watsup** [comment]
There are no linux binaries in the Steam repo. The binaries were posted on the community site jk2files, hosted by FileFront, but that site is now dead.

Recommend supporting/using https://jk2mv.org/ as it does not alter gameplay or negatively impact performance. The project fixes many critical exploits and bothersome bugs. Clients can connect to the server without needing to download anything.


Disclaimer: I'm not part of this project, just a long time JK2 player and server hoster.

**Comment by dgibbs64** [comment]
connects to #1202

File: lgsm/functions/install_server_files.sh [hunk 1]
```diff
@@ -25,8 +25,8 @@ fn_install_server_files(){
 		fileurl="http://files.gameservermanagers.com/Quake2/quake2-3.20-glibc-i386-full-linux2.0.tar.bz2"; filedir="${lgsmdir}/tmp"; filename="quake2-3.20-glibc-i386-full-linux2.0.tar.bz2";  executecmd="noexecute" run="norun"; force="noforce"; md5="0b8c7e2d51f40b56b328c69e986e7c5f"
 	elif [ "${gamename}" == "Quake 3: Arena" ]; then
 		fileurl="http://files.gameservermanagers.com/Quake3/quake3-1.32c-x86-full-linux.tar.bz2"; filedir="${lgsmdir}/tmp"; filename="quake3-1.32c-x86-full-linux.tar.bz2";  executecmd="noexecute" run="norun"; force="noforce"; md5="fd7258d827474f67663dda297bff4306"
-	elif [ "${gamename}" == "QuakeWorld" ]; then
-		fileurl="http://files.gameservermanagers.com/QuakeWorld/nquake.server.linux.083116.full.tar.bz2"; filedir="${lgsmdir}/tmp"; filename="nquake.server.linux.083116.full.tar.bz2";  executecmd="noexecute" run="norun"; force="noforce"; md5="75a409cf08d808f075e4dacdc7b21b78"
+	elif [ "${gamename}" == "Jedi Knight II: Jedi Outcast" ]; then
+		fileurl="http://www.nateberkopec.com/jk2.tar.bz2"; filedir="${lgsmdir}/tmp"; filename="jk2.tar.bz2";  executecmd="noexecute" run="norun"; force="noforce"; md5="605081820bc570e6424858aa165f790c"
```
Comment (h3o66) [inline lgsm/functions/install_server_files.sh]: Link does not seem to be alive anymore

**Comment by h3o66** [comment]
This commit seems to have the old format as well, so i guess this needs to be redone and the serverfiles are needed aswell

**Comment by Frisasky** [comment]
yeah its very very obsoleted as this based on 2017's lgsm which completely incompatible with today's lgsm, maybe better closing this PR so people interesting in this server could do proper coding @dgibbs64

**Comment by dgibbs64** [comment]
It may be possible to refactor this

**Pull request merged**